SuperBuild/python_configure_python_launcher.cmake

```diff
@@ -126,6 +126,21 @@ set(PYTHONLAUNCHER_PYTHONPATH_INSTALLED
   "<APPLAUNCHER_DIR>/../${Slicer_BIN_DIR}/Python"
   )
 
+# Add paths necessary to import VTK
+if (APPLE)
+  list(APPEND PYTHONLAUNCHER_PYTHONPATH_INSTALLED
+    "${PYTHONHOME}/lib/Slicer-4.9"
+    )
+elseif(MSVC)
+  list(APPEND PYTHONLAUNCHER_PYTHONPATH_INSTALLED
+    "${PYTHONHOME}/${Slicer_BIN_DIR}/Lib/site-packages"
```

Comment:
I think this should be:
"<APPLAUNCHER_DIR>/../${Slicer_BIN_DIR}/Lib/${PYTHON_SITE_PACKAGES_SUBDIR}"